packages/doc/src/components/Tokens/Cards.jsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import styled from 'styled-components';
+import { string, arrayOf, any, shape } from 'prop-types';
+import { hexToRgb } from '@gympass/yoga-common';
+import withToken from './withToken';
+
+const CardsWrapper = styled.section`
```

Comment:
You are repeating this `Wrapper` in `Tokens` component (Cards, Colors), I think you can save this in a component and reuse it.